### PR TITLE
Content changes for Marriage Abroad (same sex civil partnerships in Greece)

### PR DESCRIPTION
**Pull request by robinjam**
[Trello card](https://trello.com/c/2b8M8J7l/233-greece-marriage-tool-same-sex-civil-partnerships-content-change-request)
[Zendesk ticket](https://govuk.zendesk.com/agent/tickets/1375581)

## Description

Change same-sex outcomes for Greece as a result of changes to the law in Greece making civil partnerships legal.

## Factcheck

[Preview link for factcheck](https://smart-answers-pr-2708.herokuapp.com/marriage-abroad)

[Original version on gov.uk](https://www.gov.uk/marriage-abroad)

### All outcomes where user lives in UK

* https://smart-answers-pr-2708.herokuapp.com/marriage-abroad/y/greece/uk/partner_local/same_sex
* https://smart-answers-pr-2708.herokuapp.com/marriage-abroad/y/greece/uk/partner_british/same_sex
* https://smart-answers-pr-2708.herokuapp.com/marriage-abroad/y/greece/uk/partner_other/same_sex

#### Before

screen shot 2016-09-02 at 11 51 27

#### After

screen shot 2016-09-02 at 11 51 29

### All outcomes where user lives outside UK

* https://smart-answers-pr-2708.herokuapp.com/marriage-abroad/y/greece/third_country/partner_british/same_sex
* https://smart-answers-pr-2708.herokuapp.com/marriage-abroad/y/greece/third_country/partner_local/same_sex
* https://smart-answers-pr-2708.herokuapp.com/marriage-abroad/y/greece/third_country/partner_other/same_sex
* https://smart-answers-pr-2708.herokuapp.com/marriage-abroad/y/greece/ceremony_country/partner_other/same_sex
* https://smart-answers-pr-2708.herokuapp.com/marriage-abroad/y/greece/ceremony_country/partner_local/same_sex
* https://smart-answers-pr-2708.herokuapp.com/marriage-abroad/y/greece/ceremony_country/partner_british/same_sex

#### Before

screen shot 2016-09-02 at 11 51 43

#### After

screen shot 2016-09-02 at 11 51 48

### CODE REVIEW

File: lib/smart_answer_flows/marriage-abroad/outcomes/outcome_same_sex_civil_partnership_in_greece.govspeak.erb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    The local [notary public](http://www.notaries-directory.eu/) in Greece will prepare a cohabitation agreement. You’ll need to sign it with your partner in front of the notary. 
+
+    You’ll also need to get the documentation [interpreted](https://www.gov.uk/government/publications/translators-and-interpreters-list) if you don’t speak Greek. 
+
+    You’ll both need your current passports or national identity cards. 
+
+    Once the agreement has been signed, you’ll need to register the agreement with the local town hall. The registrar will then issue you with a special registration certificate.
+  <% end %>
+
+  ##Naturalisation of your partner if they move to the UK
```
Comment (leenagupte) [inline lib/smart_answer_flows/marriage-abroad/outcomes/outcome_same_sex_civil_partnership_in_greece.govspeak.erb]: There is a partial template (_partner_naturalisation_in_uk.govspeak.erb) where the wording is identical to what is written here. The only difference is that the partial links to the "Become a British citizen" page on GOV.UK.

There is a case to be made for reusing the partial here.

Comment (robinjam) [inline lib/smart_answer_flows/marriage-abroad/outcomes/outcome_same_sex_civil_partnership_in_greece.govspeak.erb]: Done

**Comment by leenagupte** [comment]
@robinjam Teeny thing. Can you keep git commit messages to 50/72 characters (see commit 1fdbeaf).

**Comment by pmanrubia** [comment]
@robinjam  / @leenagupte another option for keeping the commits atomic could be:

1. Add the outcome to the flow but copying the same content as in opposite sex.
```cp pathto/outcome_same_sex_marriage_and_civil_partnership_not_possible pathto/outcome_same_sex_civiil_partnership_in_greece```
Update the flow and keep the same content.
Clarifying in the commit notes that this is a `preparatory commit` in order to change the copy of the outcome: there should not be any changes to the artefacts (all tests should be green)
2. Do the copy changes, regenerate artefacts and commit all together. Again a gree commit.
3. Extract partial.

This way we have three atomic (and self-explanatory) commits

**Comment by leenagupte** [comment]
@pmanrubia @robinjam Where a brand new outcome is involved, I don't think it's helpful to initially create the outcome template with the wrong text in it. Personally, I would find that more confusing.

**Comment by pmanrubia** [comment]
I would prefer having atomic commits where all tests are green in every moment. This approach will help us, for example, to run `git bisect` at any time, which is always a good thing. In my opinion, the commit history would be clear with my suggestion as we are moving our codebase slowly, in smaller chunks, and always in `Green state`. And a side note, this would not be `the wrong text`, it would just be the same status of the app expressed in a different way (the new outcome)

You can find similar (but not identical) examples in:

* https://github.com/alphagov/smart-answers/pull/2688
* https://github.com/alphagov/smart-answers/pull/2653

In any case, this is your call @robinjam and @leenagupte. I don't have strong opinions on the way you have played this card.

**Comment by robinjam** [comment]
@leenagupte @pmanrubia In this particular case the change is so small that it's probably easiest if I just squash this PR down into a single commit.

**Comment by pmanrubia** [comment]
I would still prefer having commits split as described in https://github.com/alphagov/smart-answers/pull/2708#issuecomment-243489464, but that shouldn't prevent this PR from being merged.

LGTM

**Pull request merged**